src/evalfunc.c

```diff
@@ -3867,6 +3873,28 @@ f_cursor(typval_T *argvars, typval_T *rettv)
     set_cursorpos(argvars, rettv, FALSE);
 }
 
+/*
+ * "cursor_off()" function
+ *
+ * Disable cursor.
+ */
+    static void
+f_cursor_off(typval_T *argvars UNUSED, typval_T *rettv UNUSED)
+{
+    cursor_sleep();
```

Comment:
does that work in the GUI?

Reply:
It works both CUI and GUI.

![2024-04-08_09-51](https://github.com/vim/vim/assets/41495/64b3171f-202c-4bbc-9d33-39d0196a3993)